### PR TITLE
Super Marquee

### PR DESCRIPTION
**Pull request by thomas-forte**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@babel/core": "^7.16.0",
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
+    "@fontsource/dotgothic16": "^5.0.18",
```
Comment (drewgbarnes) [inline frontend/package.json:8]: huh didnt know you could get fonts via packages

File: frontend/src/components/SuperMarquee.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  return (
+    <div
+      className={
+        (easterEggMode ? styles.easterEggMode : "") +
+        " " +
+        styles.marquee +
+        " text-green-400 min-w-100 border-y-4 border-gray-400 bg-black"
+      }
+      onDoubleClick={() => setEasterEggMode(!easterEggMode)}
```
Comment (drewgbarnes) [inline frontend/src/components/SuperMarquee.tsx:26]: TIL onDoubleClick exists

File: frontend/src/components/SuperMarquee.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  const [easterEggMode, setEasterEggMode] = useState(false);
+
+  return (
+    <div
+      className={
+        (easterEggMode ? styles.easterEggMode : "") +
+        " " +
+        styles.marquee +
+        " text-green-400 min-w-100 border-y-4 border-gray-400 bg-black"
+      }
```
Comment (drewgbarnes) [inline frontend/src/components/SuperMarquee.tsx:25]: `classNames` package is kinda nice for this

**Review by drewgbarnes — APPROVED** [review]

File: frontend/package.json (hunk 1 above)
Comment (thomas-forte) [reply frontend/package.json:8]: yee, I prefer over cdn and fontsource has all the open license ones google has.

File: frontend/src/components/SuperMarquee.tsx (hunk 2 above)
Comment (thomas-forte) [reply frontend/src/components/SuperMarquee.tsx:26]: yeah I was halfway through creating a special timeout function then I saw react natively supports it, made the job easy

File: frontend/src/components/SuperMarquee.tsx (hunk 3 above)
Comment (thomas-forte) [reply frontend/src/components/SuperMarquee.tsx:25]: I'll have to look that up, Angular has a nice way to add class depending on boolean flags but stringing them along is kinda cumbersome in react

Comment (thomas-forte) [reply frontend/src/components/SuperMarquee.tsx:25]: I think what this copy pasta code from tailwind is attempting to do here, https://github.com/thomas-forte/striped-sweater/blob/0713119ed8698f7f6788a6df3b53eef6168a310d/frontend/src/components/Navbar.tsx#L28

**Pull request merged**